packages/cadl-rlc-test/cadl-output/src/parameters.ts

```diff
@@ -0,0 +1,21 @@
+import { RequestParameters } from "@azure-rest/core-client";
+import { Pet } from "./models";
+
+export type PetsDeleteParameters = RequestParameters;
+export type PetsReadParameters = RequestParameters;
+
+export interface PetsCreateBodyParam {
```

Comment:
Will the @doc decorator in CADL be the comments of those exports?

Reply:
Will enhance it in future pr